keyboards/treadstone48/rev1/split_scomm.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SPLIT_COMM_H
```

Comment:
This include guard can be replaced with `#pragma once`

```suggestion
#pragma once
```